RELEASING/README.md

```diff
@@ -22,7 +22,7 @@ under the License.
 You'll probably want to run these commands manually and understand what
 they do prior to doing so.
 
-## Release setup
+## Release setup (one-off)
```

Comment:
maybe `(First Time Only)` as opposed to `one-off`? Not sure if that's clearer...

Reply:
Yes much better.